lib/v2/screens/profile/components/profile_header.dart

```diff
@@ -23,38 +22,10 @@ class ProfileHeader extends StatelessWidget {
                 children: [
                   Column(
                     children: [
-                      ClipRRect(
-                        borderRadius: BorderRadius.circular(50.0),
-                        child: Container(
-                          width: 100.0,
-                          height: 100.0,
-                          child: state.profile?.image == null
-                              ? const SizedBox.shrink()
-                              : CachedNetworkImage(
-                                  imageUrl: state.profile?.image ?? '',
-                                  fit: BoxFit.cover,
-                                  // Show a nice loading as placeholder
-                                  // placeholder: (_, url) =>
-                                  //     Image.asset('assets/images/loading_image_placeholder.gif', fit: BoxFit.cover),
-                                  errorWidget: (context, url, error) {
-                                    return Container(
-                                      color: AppColors.getColorByString(state.profile?.nickname ?? ''),
-                                      child: Center(
-                                        child: Text(
-                                          (state.profile?.nickname != null)
-                                              ? state.profile?.nickname?.substring(0, 2)?.toUpperCase()
-                                              : '?',
-                                          style: const TextStyle(
-                                            color: Colors.white,
-                                            fontSize: 24,
-                                            fontWeight: FontWeight.w600,
-                                          ),
-                                        ),
-                                      ),
-                                    );
-                                  },
-                                ),
-                        ),
+                      ProfileAvatar(
+                        image: state.profile.image,
+                        nickname: state.profile.nickname,
+                        account: state.profile.account,
```

Comment:
hi, I think you missing to add the size here

Reply:
Nope, its there as default that you can override.

Reply:
Now size is required